app/views/foreman_leapp/job_templates/leapp_install.erb

```diff
@@ -0,0 +1,36 @@
+<%#
+kind: job_template
+name: Install Leaoo
+job_category: Leapp - Install
+description_format: 'Install leapp package'
+provider_type: Ansible
+feature: leapp_upgrade
+model: JobTemplate
+%>
+<%
+  is_rhel = @host.operatingsystem.name == 'RedHat'
+  os_major = @host.operatingsystem.major.to_i
+-%>
+---
+- hosts: all
+  tasks:
+<% if is_rhel &&  os_major == 7 -%>
+    - name: Enable rhel-7-server-rpms
+      command: subscription-manager repos --enable rhel-7-server-rpms
+    - name: Enable rhel-7-server-extras-rpms repo
+      command: subscription-manager repos --enable rhel-7-server-extras-rpms
+    - name: Install leapp
+      yum:
```

Comment:
Why `yum` or `dnf`? Can't we just use `package` here and let ansible figure it out?